apps/web-connect/src/features/checkout/components/ActionSection.tsx

```diff
@@ -34,7 +35,7 @@ export function ActionSection(): JSX.Element {
      */
     const getButtonText = useCallback(() => {
         if (mintWithEth.isLoading || mintWithXai.isLoading) return "WAITING FOR CONFIRMATION";
-        if (chain?.id !== 42161) return "Please Switch to Arbitrum One";
+        // if (chain?.id !== 42161) return "Please Switch to Arbitrum One";
```

Comment:
Same as the comment bellow.

We only would remove these checks on a sepolia testnet build.